src/app/pages/CpsSty/categoryMap/index.js

```diff
@@ -0,0 +1,21 @@
+const categoryType = category => {
+  const dictonary = {
+    News: 'ReportageNewsArticle',
+    Feature: 'ReportageNewsArticle',
+    Infographic: 'ReportageNewsArticle',
+    Interactive: 'ReportageNewsArticle',
+    'Newsbeat Investigates': 'ReportageNewsArticle',
+    Summary: 'ReportageNewsArticle',
+    'You what?': 'ReportageNewsArticle',
+    Quiz: 'ReportageNewsArticle',
+    Explainer: 'BackgroundNewsArticle',
+    Analysis: 'AnalysisNewsArticle',
+    Opinion: 'OpinionNewsArticle',
+    'Advertiser Content': 'AdvertiserContentArticle',
+    Review: 'ReviewNewsArticle',
+    'Ask the Audience': 'AskPublicNewsArticle',
+  };
+  return dictonary[category] ? dictonary[category] : 'NewsArticle';
```

Comment:
with ES2020 support in babel you should be able to shorten this.
```suggestion
  return dictonary[category] ?? 'NewsArticle';
```

Reply:
Actually, this is experimental syntax apparently, so we might have ti remove it :(

```
 This experimental syntax requires enabling the parser plugin: 'nullishCoalescingOperator' (18:29)
```